README.md

```diff
@@ -1 +1,64 @@
+
+[**Main concepts and components:**](http://www.pac4j.org/docs/main-concepts-and-components.html)
+
+1) A [**client**](http://www.pac4j.org/docs/clients.html) represents an authentication mechanism. It performs the login process and returns a user profile. An indirect client is for web application authentication while a direct client is for web services authentication:
+
+&#9656; OAuth - SAML - CAS - OpenID Connect - HTTP - OpenID - Google App Engine - LDAP - SQL - JWT - MongoDB - CouchDB - Kerberos - IP address - Kerberos (SPNEGO) - REST API
+
+2) An [**authorizer**](http://www.pac4j.org/docs/authorizers.html) is meant to check authorizations on the authenticated user profile(s) or on the current web context:
+
+&#9656; Roles / permissions - Anonymous / remember-me / (fully) authenticated - Profile type, attribute -  CORS - CSRF - Security headers - IP address, HTTP method
+
+3) The `SecurityFilter` protects an url by checking that the user is authenticated and that the authorizations are valid, according to the clients and authorizers configuration. If the user is not authenticated, it performs authentication for direct clients or starts the login process for indirect clients
+
+4) The `CallbackRoute` finishes the login process for an indirect client
+
+5) The `LogoutRoute` logs out the user from the application and triggers the logout at the identity provider level.
+
+
+## Usage
+
+1) [Add the required dependencies](https://github.com/pac4j/http4s-pac4j/wiki/Dependencies)
+
+2) Define:
+
+  - the [security configuration](https://github.com/pac4j/http4s-pac4j/wiki/Security-configuration)
+  - the [callback configuration](https://github.com/pac4j/http4s-pac4j/wiki/Callback-configuration), only for web applications
+  - the [logout configuration](https://github.com/pac4j/http4s-pac4j/wiki/Logout-configuration)
+
+3) [Apply security](https://github.com/pac4j/http4s-pac4j/wiki/Apply-security)
+
+4) [Get the authenticated user profiles](https://github.com/pac4j/http4s-pac4j/wiki/Get-the-authenticated-user-profiles)
+
+
+## Demo
+
+The demo webapp: [http4s-pac4j-demo](https://github.com/pac4j/http4s-pac4j-demo) is available for tests and implements many authentication mechanisms: Facebook, Twitter, form, basic auth, CAS, SAML, OpenID Connect, JWT...
+
+
+## Versions
+
+The latest released version is the [![Maven Central](https://maven-badges.herokuapp.com/maven-central/org.pac4j/http4s-pac4j/badge.svg?style=flat)](https://maven-badges.herokuapp.com/maven-central/org.pac4j/http4s-pac4j), available in the [Maven central repository](https://repo.maven.apache.org/maven2).
+The [next version](https://github.com/pac4j/http4s-pac4j/wiki/Next-version) is under development.
+
+See the [release notes](https://github.com/pac4j/http4s-pac4j/wiki/Release-Notes). Learn more by browsing the [pac4j documentation](http://www.pac4j.org/3.3.x/docs/index.html) and the [http4s-pac4j Javadoc](http://www.javadoc.io/doc/org.pac4j/http4s-pac4j/3.0.0).
```

Comment:
The versions need to be fixed here: pac4j -> 3.6.x, https-pac4j -> 1.0.0 (after the release)

Reply:
done